setup.sh

```diff
@@ -5,94 +5,17 @@ case `uname` in
   *) ECHO="echo" ;;
 esac
 
-usage(){
-	EXIT=$1
-	$ECHO "setup.sh [options]"
-	$ECHO
-	$ECHO "Options:"
-	$ECHO "-d              \tuse the developer branch of Coffea (default = 0)"
-	$ECHO "-h              \tprint this message and exit"
-	$ECHO "-n [NAME]       \toverride the name of the virtual environment (default = coffeaenv)"
-	exit $EXIT
-}
-
 NAME=coffeaenv
-LCG=/cvmfs/sft.cern.ch/lcg/views/LCG_101cuda/x86_64-centos7-gcc8-opt
-DEV=0
-
-# check arguments
-while getopts "dhn:" opt; do
-	case "$opt" in
-		d) DEV=1
-		;;
-		h) usage 0
-		;;
-		n) NAME=$OPTARG
-		;;
-		:) printf "missing argument for -%s\n" "$OPTARG" >&2
-		   usage -1
-		;;
-		\?) printf "illegal option: -%s\n" "$OPTARG" >&2
-		    usage -2
-		;;
-	esac
-done
-
-# Setup the LCG environment
-$ECHO "Getting the LCG environment ... "
-source $LCG/setup.sh
 
-# Install most of the needed software in a virtual environment
-# following https://aarongorka.com/blog/portable-virtualenv/, an alternative is https://github.com/pantsbuild/pex
 $ECHO "\nMaking and activating the virtual environment ... "
-python -m venv --copies $NAME
+python -m venv --system-site-packages $NAME
 source $NAME/bin/activate
 
-$ECHO "\nSetup for Dask on LPC ... "
-pypackages=lib/python3.8/site-packages/
-lcgprefix=${LCG}/${pypackages}
-# need to remove python path from LCG to avoid dask conflicts
-export PYTHONPATH=""
-ln -sf ${lcgprefix}/pyxrootd ${NAME}/${pypackages}/pyxrootd
-ln -sf ${lcgprefix}/XRootD ${NAME}/${pypackages}/XRootD
-git clone git@github.com:cms-svj/lpc_dask
-python -m pip install --no-cache-dir pip --upgrade
-python -m pip install --no-cache-dir dask[dataframe]==2020.12.0 distributed==2020.12.0 dask-jobqueue
```

Comment:
These aren't still needed?

Reply:
I haven't checked dask stuff. I was thinking reviving that would be part of another PR.